### PR TITLE
StyleTransfer lib added

### PR DESCRIPTION
**Pull request by ANSHUMAN87**

### CODE REVIEW

File: stylize/src/main/java/ohos/ai/stylize/Stylize.java [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+import org.apache.tvm.NDArray;
+import org.apache.tvm.TVMType;
+import org.apache.tvm.TVMValue;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.IntBuffer;
+
+/**
+ * Main Class for Food Classifier.
```
Comment (VinodSKumar) [inline stylize/src/main/java/ohos/ai/stylize/Stylize.java]: Comment needs to be updated

**Review by VinodSKumar — DISMISSED** [review]
The Style transfer lib seems to be OK, need to add a readme file to document the lib usage.
TVM lib Suggestion: Also we can see that there is a boiler plate code duplication in all the libs due to the interface exposed in TVM lib. Suggest to check the possibility to add an utility in TVM lib to load graph, lib, creating executor and load params.

**Comment by VinodSKumar** [comment]
OK

**Review by VinodSKumar — DISMISSED** [review]
OK

File: README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Style Transfer
+The ability to create a new image,known as a pastiche, based on two input images:one representing the artistic style and one representing the content using deep learning.
+
+
+# Source
+This library has been inspired by [TensorflowHub\\Style Transfer](https://tfhub.dev/google/lite-model/magenta/arbitrary-image-stylization-v1-256/int8/prediction/1).
+
```
Comment (sharathks84) [inline README.md:11]: Add images which will give clear picture of the feature

File: README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+```slice
+    	Stylize stylizer = new Stylize(contentImgPath, contentImgName, styleImgPath, styleImgName,
+    			getResourceManager(), getCacheDir());
+       outputImage = stylizer.getOutput();
+```
+Check the example app for more information.
+
+## License
+
```
Comment (sharathks84) [inline README.md:44]: License file is missing copyright is fine we should also add License Apache 2.0

**Review by sharathks84 — APPROVED** [review]
Ok approved

**Review by Gleelakrishna — APPROVED** [review]
ok

**Pull request merged**